api/kyverno/v1/policy_types.go

```diff
@@ -44,7 +44,7 @@ type Spec struct {
 
 	// Rules is a list of Rule instances. A Policy contains multiple rules and
 	// each rule can validate, mutate, or generate resources.
-	Rules []Rule `json:"rules,omitempty" yaml:"rules,omitempty"`
+	RawRules []Rule `json:"rules,omitempty" yaml:"rules,omitempty"`
```

Comment:
Why change the name here? This here would be a breaking change for any project which imports the kyverno project as a go dependency.

Reply:
Reverted to the original name.